python_modules/libraries/dagster-fivetran/dagster_fivetran/asset_defs.py

```diff
@@ -359,7 +361,10 @@ def __init__(
     def _get_connectors(self) -> Sequence[FivetranConnectionMetadata]:
         output_connectors: List[FivetranConnectionMetadata] = []
 
-        groups = self._fivetran_instance.make_request("GET", "groups")["items"]
+        if self._desination_ids is None:
```

Comment:
```suggestion
        if not self._destination_ids:
```
treat empty list as false too, unless an empty array actually make sense for this call?